src/redis/hyperloglog.h

```diff
@@ -23,10 +23,16 @@ struct HllBufferPtr {
   size_t size;
 };
 
+//createSizeTConfig("hll-sparse-max-bytes", NULL, MODIFIABLE_CONFIG, 0, LONG_MAX, server.hll_sparse_max_bytes, 3000, MEMORY_CONFIG, NULL, NULL),
+extern size_t HLL_SPARSE_MAX_BYTES;
```

Comment:
Todo: Make this configurable.
Could anyone give me some suggestion on how to add a configuration?

Reply:
I think that it's fine to keep as a define, at least for now.
If we'll find that users want to customize this, we'll add some flag or other means to set it.